eras/byron/crypto/cardano-crypto-wrapper.cabal

```diff
@@ -83,7 +83,7 @@ library
         canonical-json,
         cardano-ledger-binary >=1.3.1,
         cardano-crypto,
-        cardano-prelude >=0.1.0.1,
+        cardano-prelude >=0.2.0.0,
```

Comment:
Could you please increment the package version in this cabal file to `1.5.1.3` ?